tools/ipmonitor/README.md

```diff
@@ -0,0 +1,98 @@
+# Mozilla VPN - IPMonitor
+
+IPMonitor is a simple libpcap app to see which packages are sent and received
+outside the VPN tunnel. This is meant to be used as a debugging/testing tool
+and it's not included in the MozillaVPN app bundle.
+
+[![asciicast](https://asciinema.org/a/HZVQMHfQew3cm0lYUsLVZ78AT.png)](https://asciinema.org/a/HZVQMHfQew3cm0lYUsLVZ78AT)
```

Comment:
Are asciinema.org links stable? If this is just a GIF or PNG then we should commit it to this repository along with the README so that it can't be taken down or leave a broken link in the future.